e2e/example_test.go

```diff
@@ -12,13 +12,17 @@ import (
 
 func TestExamplesAreValid(t *testing.T) {
 	err := filepath.Walk("../examples", func(p string, info os.FileInfo, err error) error {
+		dockerAppDir := filepath.Join(p, filepath.Base(p)+".dockerapp")
```

Comment:
What is this for?

Reply:
For the case of the folder `examples/cnab-helm` that doesn't have a `.dockerapp` inside.
Before this patch, I would try to execute and fail.
What it does now is that it detects if there is a `docker-app` app before executing.
I would like to have another check for `cnab` folders (which is the case of this one) but I'm not sure about how.
If you know, please tell me so I can add the a new case in this test.